UnicornRender/video/source/vulkan/Buffer.cpp

```diff
@@ -31,7 +31,7 @@ bool Buffer::Create(vk::PhysicalDevice physicalDevice, vk::Device device,
 {
     if(size == 0)
     {
-        LOG_ERROR("Can't allocate zero Vulkan buffer on gpu!");
+        spdlog::get(utility::Logger::LoggerName())->error("Can't allocate zero Vulkan buffer on gpu!");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.